examples/06-verif-manual/vm-010-bending_of_a_t-shaped_beam.py

```diff
@@ -0,0 +1,299 @@
+mapdl = launch_mapdl()
+
+
+###############################################################################
+# Pre-Processing
+# ~~~~~~~~~~~~~~
+# Enter verification example mode and the pre-processing routine.
+
+mapdl.clear()
+mapdl.verify()
+_ = mapdl.prep7()
+
+
+###############################################################################
+# Define Element Type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element type ``BEAM188``.
+
+# Type of analysis: Static.
+mapdl.antype("STATIC")
+
+# Element type: BEAM188.
+mapdl.et(1, "BEAM188")
+
+# Special Features are defined by keyoptions of BEAM188:
+
+# KEYOPT(3)
+# Shape functions along the length:
+# Cubic
+mapdl.keyopt(1, 3, 3)  # Cubic shape function
+
+# Print the list with currently defined element types.
+print(mapdl.etlist())
+
+
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material, where:
+#
+# * :math:`E = 30 \cdot 10^6 psi` - Young Modulus of steel.
+# * :math:`\nu = 0.3` - Poisson's ratio.
+
+# Steel material model.
+# Define Young's moulus and Poisson ratio for Steel.
+mapdl.mp("EX", 1, 30E6)
+mapdl.mp("PRXY", 1, 0.3)
+
+# Print the list of material properties.
+print(mapdl.mplist())
+
+
+###############################################################################
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section properties for a beam elements, where:
+#
+# * :math:`w_1 = 6b = 6 * 1.5 = 9\,in` - flange width.
+# * :math:`w_2 = 2h + \frac{h}{2} = 2 * 8 + \frac{8}{2} = 20\,in` - overall depth.
+# * :math:`t_1 = \frac{h}{2} = \frac{8}{2} = 4\,in` - flange thickness.
+# * :math:`t_2 = b = 1.5\,in` - stem thickness.
+
+# Parameterization of the cross-section dimensions.
+sec_num = 1
+w1 = 9
+w2 = 20
+t1 = 4
+t2 = 1.5
+
+# Define the beam cross-section.
+mapdl.sectype(sec_num, "BEAM", "T")
+mapdl.secdata(w1, w2, t1, t2)
+
+# Print the section properties.
+print(mapdl.slist())
+
+###############################################################################
+# Define Geometry
+# ~~~~~~~~~~~~~~~
+# Set up the nodes and elements. Create nodes then create elements
+# between nodes.
+
+# Define nodes for the beam element.
+mapdl.n(1, x=0, y=0)
+mapdl.n(2, x=100, y=0)
+
+# Define one node for the orientation of the beam T-section.
+mapdl.n(3, x=0, y=1)
+
+# Print the list of the created nodes.
+print(mapdl.nlist())
+
+###############################################################################
+# Define elements
+# ~~~~~~~~~~~~~~~
+# Create element between nodes 1 and 2 using node 3 as orientational node.
+
+# Create element.
+mapdl.e(1, 2, 3)
+
+# Print the list of the elements and their attributes.
+print(mapdl.elist())
+
+# Display elements with their nodes numbers.
+cpos = [(162.20508123980457, 109.41124535475498, 112.95887397446565),
+        (50.0, 0.0, 0.0),
+        (-0.4135015240403764, -0.4134577015789461, 0.8112146563156641)]
+
+_ = mapdl.eplot(show_node_numbering=True,
```

Comment:
Mute

Reply:
Hi @germa89 
I have tried to mute the `mapdl.eplot`, but I have got an issue that `mute=True ` it is not possible to apply for the `ansys.mapdl.core.plotting.general_plotter`. 
I will leave here for now `_ = mapdl.eplot()`.
Because I have tried to apply some settings from general plotter, but it didn't help me.
May I ask you about some recommendations, please ?

Reply:
Disregard my suggestion please.